test/stats/stats_analyzer_test.cpp

```diff
@@ -0,0 +1,140 @@
+// Copyright (c) 2017 Google Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// Tests for unique type declaration rules validator.
+
+#include <string>
+#include <sstream>
+
+#include "spirv/1.1/spirv.h"
+#include "test_fixture.h"
+#include "tools/stats/stats_analyzer.h"
+
+namespace {
+
+using libspirv::SpirvStats;
+
+void FillDefaultStats(SpirvStats* stats) {
+  stats->version_hist[0x00010000] = 40;
```

Comment:
This is a little fragile.  What if more members are added to a SpirvStats.
Should this method clear stats first?  E.g
```*stats = SpirvStats();```